apps/shared/test-helpers/artifacts.js

```diff
@@ -7,5 +7,6 @@ module.exports = {
   Kernel: artifacts.require('./kernel/Kernel'),
   MiniMeToken: artifacts.require('./lib/minime/MiniMeToken'),
   StandardBounties: artifacts.require('./lib/bounties/StandardBounties'),
+  BountiesEvents: artifacts.require('./lib/bounties/BountiesEvents'),
```

Comment:
Here is the small change that enforces to import BountiesEvents on each Spoof.sol for every app 🎉 

I am sorry you experienced errors locally, it seems this one is so confusing

I blame myself because I created this "artifact-requirer" helper, but it seems is not useful anymore, my recommendation would be to move StandardBounties related code to a projects subfolder or to the integrations one, as I think it is convenient for us to get rid of test-helpers folder, I tried to follow this approach with Rewards when moving the mining functions there

Is not a big deal, but seems it is related to #588 and it actually clears lot of intermediate dead code, I think the change worths be done

I already did the changes so I will commit those, if you don't mind @topocount I just tried to clarify about the previous related comments so we can save some time in the future with these pesky things